vault/resource_ssh_secret_backend_role.go

```diff
@@ -107,6 +107,11 @@ func sshSecretBackendRoleResource() *schema.Resource {
 				Type:     schema.TypeMap,
 				Optional: true,
 			},
+			"algorithm_signer": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Default:  "ssh-rsa",
```

Comment:
I think we need to leave `algorithm_signer` as `Optional: true` and `Computed: true` and remove the default. The documentation on vaultproject.io says `ssh-rsa` is the default, but I believe that's not actually the case anymore. The API is returning `""` now and not returning any defaults